package.json

```diff
@@ -70,6 +69,8 @@
     "currency-codes": "^2.1.0",
     "date-fns": "^2.30.0",
     "ed25519-hd-key": "^1.3.0",
+    "eslint-plugin-jsx-expressions": "^1.3.1",
```

Comment:
It's better to have ESLint plugins in devDeps (so it's consistent with other eslint plugins we have)